workers/unity/Packages/com.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -27,10 +27,15 @@ internal static class GenerateCode
         private static readonly string StartupCodegenMarkerFile =
             Path.GetFullPath(Path.Combine("Temp", "ImprobableCodegen.marker"));
 
+        private static readonly Regex dotnetRegex = new Regex(
+            @"(?<file>[\w\\\.]+)\((?<line>\d+),(?<col>\d+)\): (?<type>\w+) (?<code>\w+): (?<message>[\s\S]+)",
```

Comment:
May be useful to write out a string literal example of the message format in a comment above

Reply:
Yes please - anywhere there's a non-trivial Regex it needs really good documentation.

Reply:
I can add one, but its rather long and will break our formatting rules, is that ok?